safe_transaction_service/history/tests/test_safe_events_indexer.py

```diff
@@ -216,8 +221,12 @@ def test_safe_events_indexer(self):
         multisig_tx = safe.build_multisig_tx(safe_address, 0, data)
         multisig_tx.sign(owner_account_1.key)
         multisig_tx.execute(self.ethereum_test_account.key)
+        blocks_processed = (
+            self.safe_events_indexer.ethereum_client.current_block_number
+            - current_block
+        )
         # Process events: SafeMultiSigTransaction, AddedOwner, ExecutionSuccess
-        self.assertEqual(self.safe_events_indexer.start(), 3)
+        self.assertEqual(self.safe_events_indexer.start(), (3, blocks_processed))
```

Comment:
I would hardcode the number of expected blocks

Reply:
Can't be harcoded everywhere because some tests depends the ganache state (last block)